lib/cisco_node_utils/cmd_ref/vpc.yaml

```diff
@@ -50,16 +50,19 @@ domain:
 dual_active_exclude_interface_vlan_bridge_domain: 
   kind: string
   N7k:
-    get_value: '/^dual\-active exclude interface\-vlan\-bridge\-domain\s+(\S+)/'
+    get_value: '/^dual-active exclude interface-vlan-bridge-domain\s+(\S+)/'
     set_value: "<state> dual-active exclude interface-vlan-bridge-domain <range>"
   else:
-    get_value: '/^dual\-active exclude interface\-vlan\s+(\S+)/'
+    get_value: '/^dual-active exclude interface-vlan\s+(\S+)/'
     set_value: "<state> dual-active exclude interface-vlan <range>"
   default_value: 'none'
 
 fabricpath_emulated_switch_id: 
   _exclude: [N3k, N5k, N6k, N9k]
   kind: int
+  get_command:  'show running-config vpc all'
+  get_context:  ['/^vpc domain\s*(\d+)$/']
```

Comment:
Not needed, it is my lack of understanding of how template works. I thought if either set_context or get_context is brought to the property level, then the template couldn't be used at all for the property.